MyStore/MyStore.cpp

```diff
@@ -1,12 +1,38 @@
-﻿#include "JsonParcer.h"
+﻿#include "StorageData.h"
+#include <conio.h>
+#include <chrono>
 
+const int period_for_element_milliseconds = 1;
+using namespace std::chrono;
 
-int main(int argc, char *argv[])
+std::ostream& operator << (std::ostream& out, const InfoProduct& data)
 {
+	out << "Name:" << data.m_product_color << '\n' 
+		<< "Color:" << data.m_product_name << '\n';
+	return out;
+}
 
-	JsonParcer obj("..\\MyStore\\data\\Data.json");
-	obj.Deserialization();
-	obj.ShowData();
+
+int main(int argc, char* argv[])
+{
+	StorageData storage;
+
+	storage.m_Parcer.Deserialization();
+	storage.m_Parcer.ShowData();
+
+	auto last_time = system_clock::now();
+
+	while (true)
+	{
+		if ((duration_cast<seconds>(system_clock::now() - last_time).count()) > period_for_element_milliseconds)

```

Comment:
You mixed seconds and milliseconds here. Cast is done for seconds, while constant is in milliseconds. Please make them consistent.